src/api/cast.ts

```diff
@@ -8,17 +8,19 @@ import { formatCasts } from '../lib/utils.js'
  * Insert casts in the database
  * @param msg Hub event in JSON format
  */
-export async function insertCasts(msgs: Message[]) {
+export async function insertCasts(msgs: Message[]): Promise<void> {
```

Comment:
I'm generally in the camp of inferring return types where possible, in which case this feels overly verbose. Most of the reasoning is here https://www.youtube.com/watch?v=I6V2FkW1ozQ

I can be convinced otherwise though